guides/manual-installation.md

```diff
@@ -39,29 +45,31 @@ Enable PackageHub repository:
 SUSEConnect --product PackageHub/15.5/x86_64
 ```
 
-Add the prometheus user/group:
+> Note: Using a different Service Pack then SP5 requires to change repository: [SLE15 SP3: `SUSEConnect --product PackageHub/15.3/x86_64`, SLE15 SP4: `SUSEConnect --product PackageHub/15.4/x86_64`]
+
+Add the prometheus user/group **(ONLY FOR SLE15 SP5)**:
 
 ```bash
 groupadd --system prometheus
 useradd -s /sbin/nologin --system -g prometheus prometheus
 ```
 
-After a successful registration, you can install prometheus using zypper:
+Install prometheus using zypper:
 
 ```bash
 zypper in golang-github-prometheus-prometheus
 ```
 
-We will be prompted that a missing dependency can't be satisfied:
+Missing dependency can't be satisfied **(ONLY FOR SLE15 SP5)**:
```

Comment:
I noticed while adding support for SP4 in the Ansible playbooks that SP4 also requires the Prometheus workaround. Could you change this to also include SLES15 SP4?

Reply:
Another minor thing, could you add this commit or something similar?: https://github.com/trento-project/docs/commit/6950503017540f6eaccc94870008df2d56212ba1 see task TRNT-2689